Test/Common/DomainTest/implementation/VolunteerOpportunityTest.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Ttu.Domain;
+
+namespace Ttu.DomainTest.implementation
+{
+    [TestClass]
+    public class VolunteerOpportunityTest
+    {
+        private VolunteerOpportunity _volunteerOpportunity;
```

Comment:
This underscore prefix is reminiscent of C++, and you will see some examples produced by MS employees that follow this convention. The 'official' MS guidelines specify only camelCasing or PascalCasing.

https://msdn.microsoft.com/en-us/library/ms229043(v=vs.110).aspx